### PR TITLE
UI additionals 6

### PR DESCRIPTION
**Pull request by rolandlor**

### CODE REVIEW

**Review by ottpeter — APPROVED** [review]

File: src/MainRouter.tsx [hunk 1]
```diff
@@ -419,6 +420,21 @@ const MainRouter = (): ReactElement => {
...
+    const noRedirectedPaths = [
+      ROUTES.WELCOME1,
+      ROUTES.WELCOME2,
+      ROUTES.WELCOME3,
+      ROUTES.WELCOME4,
+      ROUTES.TACONBOARDING,
+      ROUTES.PROFILECREATION,
+    ];
+    if (!privKey && !noRedirectedPaths.includes(location.pathname as ROUTES)) {
+      navigate(ROUTES.APP); // Átirányítás az alap URL-re, ha nincs privKey
```
Comment (bosi95) [inline src/MainRouter.tsx:434]: use english

Comment (bosi95) [reply src/MainRouter.tsx:434]: maybe add username also

File: src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -33,11 +33,20 @@ const Dropdown: React.FC<DropdownProps> = ({
       <div className={clsx("dropdown__button", { dropdown__open: isOpen })}>
         <div>
           {items[activeItem]}
-          <span className="dropdown__button__subtext">{" (of 9)"}</span>
+          {/* <span className="dropdown__button__subtext">{" (of 9)"}</span> */}
```
Comment (bosi95) [inline src/components/Dropdown/Dropdown.tsx:36]: remove comment if not needed

**Review by bosi95 — APPROVED** [review]
don't forget to update package.json and lock file with the latest comment version

File: src/MainRouter.tsx (hunk 1 above)
Comment (rolandlor) [reply src/MainRouter.tsx:434]: A username name is generated at start of the application

**Pull request merged**